cmake/rlImGuiConfig.cmake.in

```diff
@@ -0,0 +1,3 @@
+@PACKAGE_INIT@
+
+include("${CMAKE_CURRENT_LIST_DIR}/rlImGuiTargets.cmake")
```

Comment:
What file does this reference? it's not in the PR.
Could you explain what the purpose of this file is?

Reply:
Target files are created automatically by cmake on executing `cmake --target install`

Reply:
Files `.cmake.in` are templates. When we call `cmake install` cmake replaces @VAR@ with the value VAR from CMakeLists.txt and the result is what the consumer of the library will get after the installation.